module-services/service-desktop/ServiceDesktop.cpp

```diff
@@ -34,14 +35,17 @@
 ServiceDesktop::ServiceDesktop()
     : sys::Service(service::name::service_desktop, "", sdesktop::service_stack),
       btMsgHandler(std::make_unique<sdesktop::bluetooth::BluetoothMessagesHandler>(this)),
-      notificationsClearTimer{
-          sys::TimerFactory::createSingleShotTimer(this,
-                                                   sdesktop::notificationsClearTimerName,
-                                                   sdesktop::notificationsClearTimerDelayMs,
-                                                   [this](sys::Timer & /*timer*/) { notificationEntries.clear(); })}
+      notificationsClearTimer{sys::TimerFactory::createSingleShotTimer(this,
+                                                                       sdesktop::notificationsClearTimerName,
+                                                                       sdesktop::notificationsClearTimerDelayMs,
+                                                                       [this](sys::Timer & /*timer*/) {
+                                                                           notificationEntries.clear();
+                                                                           notificationCurrentUid = 0;
```

Comment:
this deserves a function. Preferably in NotificationsEntries ( or whatever named - I'm not very good at naming, please follow your intuition) class :)

Reply:
Created a method in a separated class.